optimize/backend/src/main/java/io/camunda/optimize/dto/zeebe/process/ZeebeProcessInstanceDataDto.java

```diff
@@ -115,6 +116,11 @@ public List<Integer> getCallingElementPath() {
     return callingElementPath;
   }
 
+  @Override
+  public Set<String> getTags() {
+    return Set.of();
```

Comment:
🔧 Gut feeling says that `tag` information would be useful in Optimize, but I would leave it to an engineer with Optimize background to provide the best feedback here.

Reply:
I agree, but right now I try to keep the PRs at a minimal scope 😄 we could add a follow up task to the epic later.